bsk_rl/envs/general_satellite_tasking/simulation/dynamics.py

```diff
@@ -841,12 +841,15 @@ def _set_instrument(
             self.task_name, self.instrument, ModelPriority=priority
         )
 
-    @default_args(dataStorageCapacity=20 * 8e6, storageUnitValidCheck=True)
+    @default_args(
+        dataStorageCapacity=20 * 8e6, storageUnitValidCheck=True, setStorageInit=0
+    )
     def _set_storage_unit(
         self,
         dataStorageCapacity: int,
         priority: int = 699,
         storageUnitValidCheck: bool = True,
+        setStorageInit=0,
```

Comment:
Let's add a type hint (i.e. `storageInit: int=0`) and change the name to follow the convention of not including `set` in the variable names.

Reply:
Ok!